gh-136355: Deprecate `-b` and `-bb` CLI flags in 3.15 (#136363)
python/cpython: Doc/using/cmdline.rst

   .. versionchanged:: 3.5
      Affects also comparisons of :class:`bytes` with :class:`int`.

>    .. deprecated:: 3.15

Reviewer: ```suggestion
   .. deprecated-removed:: 3.15 3.17
```
I think we can still say removed, since we will remove it from the docs, no?

Author: I think that `removed` means that we will remove the thing itself. I would prefer to highlight that it is not going anywhere and avoid any possible confusion.
We can decide whether it worth having this in the docs alone later :)

Reviewer: > I think that removed means that we will remove the thing itself.

We will remove all functionality.

Author: Let's see what others think. I am ok with both, just want the intent to be clear :)

Reviewer 2: I don't think that we can use remove here. Usually remove means the actually remove the option.

Reviewer 3: Agreed with Victor.

>    .. deprecated-removed:: 3.15 3.17

Reviewer 2: The markup is not correct, no removal is scheduled.

   .. versionchanged:: 3.5
      Affects also comparisons of :class:`bytes` with :class:`int`.

   .. deprecated-removed:: 3.15 3.17

>       Deprecate :option:`-b` and :option:`-bb`

Reviewer 2: ```suggestion
      Deprecate :option:`-b` and :option:`!-bb`
```

>       Deprecate :option:`-b` and :option:`-bb`

Reviewer 2: There is a warning about unknown option `-bb`:

```suggestion
      Deprecate :option:`-b` and :option:`!-bb`
```

Author: Yes, thanks! I just synced my branch before some refactorings :)

      and schedule them for removal in Python 3.17.

>       They were mainly a transition helpers for Python2 -> Python3 era.
>       In 3.17 no :exc:`BytesWarning` won't be raised for these cases.
>       If you want to check for the same things in the future,
>       use any type-checker of your choice.

Reviewer 4: Based on MAL and Stan's good suggestions:
```suggestion
      These were primarily helpers for the Python 2 -> 3 transition.
      Starting with Python 3.17, no :exc:`BytesWarning` will be raised
      for these cases; use a type checker instead.
```

> 

Reviewer 3: Suggested improved wording:

```
      and schedule them for removal in Python 3.17.
      They were mainly meant as transition helpers for Python2 -> Python3 era.
      Starting with 3.17, the :exc:`BytesWarning` will be dropped for these cases.
      If you want to check for the same things in the future,
      please use a type-checker of your choice.
```

Reviewer: I suggest slightly simpler;

```
      and schedule them for removal in Python 3.17.
      These were primarily helpers for the Python2 -> Python3 transition.
      Starting with 3.17, no :exc:`BytesWarning` will be raised for these cases.
      If you want to check for the same things in the future, use a type-checker.
```
